From 9870f886fcad757af2a496d8de9612c95abad7ff Mon Sep 17 00:00:00 2001
From: Dima Kogan <dima@secretsauce.net>
Date: Mon, 24 Mar 2014 22:03:04 +0000
Subject: [PATCH 3/6] I now build libobjc ONLY if $(with_common_libs)

This was commented-out without a clear explanation. Without knowing more, this
certainly seems reasonable
---
 rules.defs | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/rules.defs b/rules.defs
index 386cacc..da5ce1a 100644
--- a/rules.defs
+++ b/rules.defs
@@ -899,9 +899,9 @@ ifeq ($(with_objc),yes)
     with_objcdev := yes
   endif
   # libobjc soname change in 4.7
-  #ifeq ($(with_common_libs),yes)
+  ifeq ($(with_common_libs),yes)
     with_libobjc := yes
-  #endif
+  endif
 
   enabled_languages += objc
 endif
-- 
2.0.0

